Navigation complete from show to index and show to update

Charlie19255 opened the pull request.

Charlie19255 committed:
Navigation complete from show to index and show to update

elysemontano commented on app/javascript/components/pages/ApartmentShow.js:
```diff
@@ -49,6 +48,12 @@ const ApartmentShow = ({ apartments }) => {
+            <NavLink to={`/apartmentindex`}>
+                <Button>Back to apartments</Button>
+              </NavLink>
+              <NavLink to={`/apartmentedit/`}>
```
Keep in mind that edit requires a parameter in the URL to route to.

elysemontano reviewed (changes requested):
Left a comment in ApartmentShow.js

Charlie19255 committed:
Added the string interpolation

I communicated with Blake and considered it would update once the merge was complete and they did their pull request after. Changes done
